doc/classes/TextServerDummy.xml

```diff
@@ -1,8 +1,22 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="TextServerDummy" inherits="TextServerExtension" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
+		A dummy text server that can't render text.
```

Comment:
```suggestion
		A dummy text server that can't render text, and load/import fonts.
```
Therefor, it's unusable for headless editor when imports update is required (unlike dummy audio driver, for example).

It was added to have better build granularity, and prevent crashes/error spam when building with all other text servers disabled, without any practical use case for the server itself in mind.